components/src/core/Drawer/DrawerNavItem.tsx

```diff
@@ -145,8 +145,8 @@ export const DrawerNavItem: React.FC<DrawerNavItem> = (props) => {
     const expandIcon = navItem.expandIcon || navGroupProps.expandIcon || (depth ? <ArrowDropDown /> : <ExpandMore />);
     const hidePadding = navItem.hidePadding !== undefined ? navItem.hidePadding : navGroupProps.hidePadding;
     const InfoListItemProps = navItem.InfoListItemProps || navGroupProps.InfoListItemProps || {};
-    const itemFontColor = navItem.itemFontColor || navGroupProps.itemFontColor || gray[500];
-    const itemIconColor = navItem.itemIconColor || navGroupProps.itemIconColor || gray[500];
+    const itemFontColor = navItem.itemFontColor || navGroupProps.itemFontColor || black[500];
```

Comment:
we keep switching between gray 500 and black 500....we need to pick something and stick with it.